include/zephyr/dt-bindings/clock/gd32l23x-clocks.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022 BrainCo.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_GD32L23X_CLOCKS_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_GD32L23X_CLOCKS_H_
+
+#include "gd32-clocks-common.h"
+
+/**
+ * @name Register offsets
+ * @{
+ */
+
+#define GD32_AHB1EN_OFFSET       0x14U
+#define GD32_APB1EN_OFFSET       0x1CU
+#define GD32_APB2EN_OFFSET       0x18U
+
+/** @} */
+
+/**
+ * @name Clock enable/disable definitions for peripherals
+ * @{
+ */
+
+/* AHB1 peripherals */
+#define GD32_CLOCK_DMA        GD32_CLOCK_CONFIG(AHB1EN, 0U)
+#define GD32_CLOCK_SRAM0      GD32_CLOCK_CONFIG(AHB1EN, 2U)
+#define GD32_CLOCK_FMC        GD32_CLOCK_CONFIG(AHB1EN, 4U)
+#define GD32_CLOCK_CRC        GD32_CLOCK_CONFIG(AHB1EN, 6U)
+#define GD32_CLOCK_SRAM1      GD32_CLOCK_CONFIG(AHB1EN, 7U)
+#define GD32_CLOCK_GPIOA      GD32_CLOCK_CONFIG(AHB1EN, 17U)
+#define GD32_CLOCK_GPIOB      GD32_CLOCK_CONFIG(AHB1EN, 18U)
+#define GD32_CLOCK_GPIOC      GD32_CLOCK_CONFIG(AHB1EN, 19U)
+#define GD32_CLOCK_GPIOD      GD32_CLOCK_CONFIG(AHB1EN, 20U)
+#define GD32_CLOCK_GPIOF      GD32_CLOCK_CONFIG(AHB1EN, 22U)
+
```

Comment:
It seems not to define AHB2-related definitions.
Is it your intent?

https://github.com/zephyrproject-rtos/hal_gigadevice/pull/29/files#diff-5ac2452bfb4597667a852e7df7626f3b49421c625cb82b0c7654590018f74622R292-R293

Reply:
Added, thanks!